src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,10 +99,15 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/infrastructure":
                     success = SyncToBranch(branchString, result);
                     break;
+                case "origin/release/1.0.0-rc2":
+                    context.Response.Write(string.Format("Branch name specified, '{0}' will not be synced by this service<br/>", HttpUtility.HtmlEncode(branchString)));
+                    success = true; 
```

Comment:
Correct. Whatever in RC1 or RC2 should just use whatever testing mechanism when they were forked. @iamjasonp please make sure this is the case.